core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -259,6 +259,8 @@
     
     public static final String SETSOFGENES_LOCATION = "querypage.setsofgenes.location";
 
+    public static final String WHOLE_SLIDE_VIEWER_SECRET_KEY = "whole_slide_viewer_secret_key";
```

Comment:
i believe this should use dot notation.  we convert to underscores when writing these to javascript (jsp)

Reply:
Changed to "viewer.key"